src/components/sidenavigation/SideNavigation.module.css

```diff
@@ -3,6 +3,26 @@
     justify-content: center;
 }
 
+.content {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    flex: 1 1 auto;
+    max-width: 1344px;
+}
+
+@media (max-width: 1440px) {
+    .content {
+        padding: 0 3rem;
+    }
+}
+
+@media (max-width: 1024px) {
+    .content {
+        padding: 0 1rem;
+    }
+}
```

Comment:
dette er for det meste bare litt hacking for å få sidebaren til line opp med brødsmulene i dekoratøren